test/unit/range/views/view_zip_test.cpp

```diff
@@ -0,0 +1,32 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <deque>
+#include <iostream>
+#include <list>
+#include <string>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+#include <seqan3/range/views/zip.hpp>
+#include <seqan3/std/algorithm>
+#include <seqan3/std/ranges>
+
+// https://github.com/ericniebler/range-v3/issues/1480
+TEST(zip_view, gcc10bug_rangev3_1480)
+{
+    std::vector<char> const first_sequence{};
+    std::vector<char> const second_sequence{};
+
+    auto zip_view = seqan3::views::zip(first_sequence, second_sequence);
+    auto fn = [&] ([[maybe_unused]] auto && ch)
```

Comment:
```suggestion
    auto fn = [&] ([[maybe_unused]] auto && ch)
```
Why is `ch` maybe_unused? Is that part of the test?